examples/.avocado.hint.example

```diff
@@ -0,0 +1,29 @@
+# This is just a example file. Insert this file inside your project root
```

Comment:
There's a problem with this file name, because it won't be seen by most users. I'd recommend creating a directory to host it, and a README file saying that it should be named `.avocado.hint` with emphasis on the *dot*.

Reply:
@clebergnu do you mean `examples/hint/README` and `examples/hint/.avocado.hint` ?

Actually I was planning to create an entire section on our documentation and remove this example file. What do you think?

Reply:
Yes, that may be even better.  If users are able to copy and paste (I mean, the right formatting is used) then extra points to you! :)

Reply:
I like the idea of having the template or example file and the documentation. Is it possible to keep both? Is there a downside in making this file visible in the project instead of using an invisible file? I mean, usually, configuration files are visible when they are built by the user.

Reply:
@willianrampazzo keeping both, documentation and example files, might add some extra effort to keep both updated. But maybe we can have an "include" solution, that will use the same file for both.

I will suggest keeping this file as it is, and when the new documentation is added we improve this.

Reply:
OKay, +1 for the include solution from my side too!